ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -984,7 +984,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node \n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export NODE_OPTIONS=--openssl-legacy-provider\nexport BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node \n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `NODE_OPTIONS=--openssl-legacy-provider` is a workaround for the Node.js issue where the default OpenSSL library was changed in Node.js 17.0.0, causing some packages to fail. This change forces Node.js to use the old OpenSSL library. However, this is a temporary fix and should be removed once all dependencies are updated to work with the new OpenSSL library.
```diff
- export NODE_OPTIONS=--openssl-legacy-provider
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->